.vsts-ci.yml

```diff
@@ -94,19 +94,19 @@ jobs:
       matrix:
```

Comment:
Default version should be bumped here as well.
https://github.com/microsoft/LightGBM/blob/3654ecaaa1a33628dd2b1cc936c3f1cfd31079a4/.vsts-ci.yml#L13

Reply:
ah missed this one, thanks! Updated in https://github.com/microsoft/LightGBM/pull/6668/commits/422be6a4164ad657a1d783959a5c2d1ad07825dd